Mathlib/Data/Sum/LE.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2024 Martin Dvorak. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Dvorak
+-/
+import Mathlib.Order.Basic
+import Mathlib.Algebra.Group.Pi.Basic
+/-!
+# TODO
```

Comment:
Please write a module docstring. E.g., move the next sentence to this line, then write 1 more sentence explaining what kind of lemmas do you prove here.